SPECS/python-markdown/python-markdown.spec

```diff
@@ -49,7 +49,7 @@ PYTHONPATH=%{buildroot}%{python3_sitelib} \
   LICENSE.md > LICENSE.html
 
 %check
-%{__python3} ./setup.py test
+python3 -m unittest discover -v
```

Comment:
We should keep the macro from the other side.

Reply:
fixed